src/lib/sticky-header/sticky-header.ts

```diff
@@ -97,7 +111,8 @@ export class CdkStickyHeader implements OnDestroy, AfterViewInit {
   constructor(element: ElementRef,
               scrollable: Scrollable,
               @Optional() public parentRegion: CdkStickyRegion,
-              platform: Platform) {
+              platform: Platform,
+              @Inject(STICKY_HEADER_SUPPORT_STRATEGY) public _stickyPositionSupportCheck) {
```

Comment:
Can use `_isPositionStickySupported` here as variable name and then you can remove line 155